src/Features/Core/Portable/ConvertIfToSwitch/AbstractConvertIfToSwitchCodeRefactoringProvider.cs

```diff
@@ -105,10 +105,6 @@ public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContex
             if (!sections.Any(section => GetSwitchArmKind(section.Body) == OperationKind.Return))
                 return false;
 
-            // All arms must be convertible to a switch arm
-            if (!sections.All(section => GetSwitchArmKind(section.Body) != default))
-                return false;
-
```

Comment:
this check is redundant with code below. it was called out in PR but i forgot to submit the change.